src/Shell/Task/FixtureTask.php

```diff
@@ -175,7 +176,13 @@ public function bake($model, $useTable = null)
             );
         }
         $schemaCollection = $connection->schemaCollection();
-        $data = $schemaCollection->describe($useTable);
+        try {
+            $data = $schemaCollection->describe($useTable);
+        } catch (Exception $e) {
```

Comment:
I am. `Schema/Collection::describe()` throws a `Cake\Database\Exception`, which is what I'm catching here.